.github/workflows/jest.yml

```
@@ -9,29 +10,35 @@ on:
       - "yarn.lock"
       - "esbuild.config.js"
       - "jest.config.js"
+
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup Node.js
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)